src/puppetlabs/trapperkeeper/services/webserver/jetty9_service.clj

```diff
@@ -26,23 +33,38 @@
 (defservice jetty9-service
   "Provides a Jetty 9 web server as a service"
   WebserverService
-  [[:ConfigService get-in-config]]
+  {:required [ConfigService]
+   :optional [FilesystemWatchService]}
   (init [this context]
         (log/info (i18n/trs "Initializing web server(s)."))
-        (let [config (or (get-in-config [:webserver])
+        (let [config-service (get-service this :ConfigService)
+              config (or (tk-config/get-in-config config-service [:webserver])
                          ;; Here for backward compatibility with existing projects
-                         (get-in-config [:jetty])
+                         (tk-config/get-in-config config-service [:jetty])
                          {})]
           (config/validate-config config)
           (core/init! context config)))
 
   (start [this context]
          (log/info (i18n/trs "Starting web server(s)."))
-         (let [config (or (get-in-config [:webserver])
+         (let [config-service (get-service this :ConfigService)
+               config (or (tk-config/get-in-config config-service [:webserver])
                           ;; Here for backward compatibility with existing projects
-                          (get-in-config [:jetty])
-                          {})]
-           (core/start! context config)))
+                          (tk-config/get-in-config config-service [:jetty])
+                          {})
+               started-context (core/start! context config)]
+           (if-let [filesystem-watcher-service
+                    (maybe-get-service this :FilesystemWatchService)]
+             (let [watcher (watch-protocol/create-watcher filesystem-watcher-service)]
+               (doseq [server (:jetty9-servers started-context)]
+                 (when-let [ssl-context-factory (-> server
+                                                    second
+                                                    :state
+                                                    deref
+                                                    :ssl-context-factory)]
```

Comment:
Awesome, thank you both! Made much more sense when I learned you didn't have to supply the parens.. I hadn't seen it without them so I read this as a single function call instead of the multiple calls I expected. Got it now.